Firebase/InstanceID/FIRInstanceIDKeychain.m

```diff
@@ -16,14 +16,10 @@
 
 #import "FIRInstanceIDKeychain.h"
```

Comment:
This might be some thing I can help in another PR but I wonder if FIRInstanceIDKeychain is needed at all after FIS is in.

Reply:
As far as I can see it is still used by `FIRInstanceIDAuthKeyChain` which is used in the `FIRInstanceIDTokenManager`, `FIRInstanceIDCheckinStore` and couple more classes.

Reply:
I see, so it's used to store token and checkin, not just IID. We can deal with that later then. Thanks!